### ISSUE TITLE
[Bug]: VSCode extension cannot start test-server because it always lookup config file from root workspace

### ISSUE THREAD

**Issue by iporsut** [microsoft/playwright#30870]
### Version

1.1.2

### Steps to reproduce

If you have a project structure like this

```
root-project
- api
- web
- e2e
  - playwright.config.ts
  - tests
    - test.spec.ts
```

then start to test or debug.

### Expected behavior

It should run test without any error

### Actual behavior

I have got this error.

```
/Users/weerasak/.asdf/shims/node ./e2e/node_modules/@playwright/test/cli.js test-server -c playwright.config.ts
Uncaught Error Error: [root workspace path]/playwright.config.ts does not exist
    at resolveConfigFile ([root workspace path]/e2e/node_modules/playwright/lib/common/configLoader.js:244:52)
```

### Environment

```shell
System:
    OS: macOS 14.4.1
    CPU: (8) arm64 Apple M1
    Memory: 42.88 MB / 16.00 GB
  Binaries:
    Node: 18.19.0 - ~/.asdf/installs/nodejs/18.19.0/bin/node
    Yarn: 1.22.21 - ~/.asdf/installs/nodejs/18.19.0/bin/yarn
    npm: 10.2.3 - ~/.asdf/plugins/nodejs/shims/npm
    bun: 1.0.33 - ~/.bun/bin/bun
  IDEs:
    VSCode: 1.89.1 - /usr/local/bin/code
  Languages:
    Bash: 5.2.26 - /opt/homebrew/bin/bash
  npmPackages:
    @playwright/test: ^1.44.0 => 1.44.0
```

**Comment by royalsfankc** [comment]
Just noting I am seeing the same issue today, I thought I was losing my mind, I swear it was working yesterday but maybe something updated with the test plugin overnight.

I just downgraded the plugin to 1.1.1 and things are working again.

**Comment by iporsut** [comment]
Seem like it's already fixed in 1.1.3
